ngraph_bridge/ngraph_encapsulate_clusters.cc

```diff
@@ -328,15 +328,18 @@ Status EncapsulateClusters(
           NodeBuilder::NodeOut(graph->FindNodeId(src_node_id), src_output_idx));
     }
 
+    auto config_map =
+        BackendManager::GetBackendAttributeValues(cluster_backend);
     Node* n;
-    NodeBuilder nb = NodeBuilder(ss.str(), "NGraphEncapsulate")
-                         .Attr("ngraph_cluster", cluster_idx)
-                         .Attr("ngraph_backend", cluster_backend)
-                         .Attr("Targuments", input_types)
-                         .Attr("Tresults", cluster_output_dt_map[cluster_idx])
-                         .Attr("ngraph_graph_id", graph_id)
-                         .Device(device_name_map[cluster_idx])
-                         .Input(inputs);
+    NodeBuilder nb =
+        NodeBuilder(ss.str(), "NGraphEncapsulate")
+            .Attr("ngraph_cluster", cluster_idx)
+            .Attr("ngraph_backend", config_map.at("ngraph_backend"))
```

Comment:
Change the name of the variable `config_map` since we already have `device_config` make it confusing.